### DISCUSSION TITLE
Dark Mode Auto Not Working

### DISCUSSION THREAD

**Discussion by ersingo** [bootscore/bootscore#506] [Q&A] [answered]
Hi Basti;

I bought dark mode. I used the plugin but the auto mode is not working. It's dark right now but the website is still bright :)

Is this an error? I live in the UTC+3 time zone. Web hosting is in my country. And my website settings are also set to UTC+3.

**Comment by crftwrk** [comment]
Hi Ersin,

I need a link to your site to check. Are you sure that you have enabled auto in your computer system?

**Reply by ersingo** [reply]
https://birikibilgi.com/

I'm experimenting here. I don't have this setting on my computer system. Does it do it automatically according to the computer I'm using? I did not know that.

Is it possible to set a certain time?

**Reply by crftwrk** [reply, marked as answer]
Your Dark Mode Auto works well. Auto mode checks `@media (prefers-color-scheme: [mode])` in users OS settings:


<img width="390" alt="Bildschirmfoto 2023-06-14 um 18 30 48" src="https://github.com/bootscore/bootscore/assets/51531217/3380c1da-0850-4313-8cf8-1f5acd08d1e8">


- OS setting light > Plugin light
- OS setting dark > Plugin dark
- OS setting auto > Computer shows light at daytime and dark in the night. Plugin follows these settings if Auto is enabled.

> Is it possible to set a certain time?

Not in a simple way and this would go far behind the scope. It seems to be very complicated to code and makes no sense. You have to code a time checker and this checker should then trigger the toggler twice a day. This must be done for all time zones with a location tracker, otherwise somewhere in Japan switches to night mode at daytime.

Plugin does exactly what it should do. Following the users OS settings and allows to override them by choosing light or dark in the dropdown.

> I don't have this setting on my computer system.

I'm sure you have. I use Apple, but here are [settings for Windows](https://support.microsoft.com/en-us/windows/change-colors-in-windows-d26ef4d6-819a-581c-1581-493cfcc005fe#:~:text=in%20Dark%20mode-,Select%20Start%20%3E%20Settings,Select%20Personalization%20%3E%20Colors.&text=In%20the%20list%20for%20Choose%20your%20mode%2C%20select%20Dark.). If not, check it on your phone, all phones have this settings. 

Hope that clears it up.

**Reply by justinkruit** [reply]
A lot of OSes have auto dark mode, turning on the OS dark mode at night which will automatically trigger the plugin's version as well.

It's certainly possible to write the time check etc with javascript, but that will require custom work.
